src/requirements.txt

```diff
@@ -1,27 +1,21 @@
 beautifulsoup4==4.11.1
-black>=20.8b1
-bokeh==2.3.3
-chart_studio==1.1.0
-coverage==6.2
-cryptography==37.0.4
+black==22.8.0
```

Comment:
it's probably worth putting a comment at the top of this file noting that other python modules are added by the installer script.